src/controller.h

```diff
@@ -176,6 +176,7 @@ class Controller final : public QObject {
 
   QTimer m_connectingTimer;
   QTimer m_handshakeTimer;
+  QTimer m_gleanControllerUpTimer;
```

Comment:
I have a better solution. Tell me what you think about it:
Can you move this timer in `telemetry.h/cpp`? https://github.com/mozilla-mobile/mozilla-vpn-client/blob/main/src/telemetry.h

In the end, this timer can live everywhere and I've already started a cleanup of controller.h/cpp.
If you do that, then activating and deactivating the timer when we are in StateOn/Off can be easily done.